src/services/dealer.ts

```diff
@@ -13,8 +13,16 @@ import { DealerPromotion } from "../types/models/dealerPromotion"
 import { WithValidationError } from "../types/withValidationError"
 import { withTokenRefresh } from "../tokenRefresh"
 
-export const fetchDealer = async (id: number): Promise<Dealer> => {
-  return fetchPath(Service.DEALER, `dealers/${id}`)
+import toQueryString from "../lib/toQueryString"
+
+export const fetchDealer = async (
+  id: number,
+  options: { language?: "de" | "en" | "fr" | "it" } = {}
+): Promise<Dealer> => {
+  return fetchPath(
+    Service.DEALER,
+    `dealers/${id}${options.language ? `?${toQueryString(options)}` : ""}`
```

Comment:
Can toQueryString handle empty objects? If yes, you can remove `options.language ?`

Reply:
Negative, we would then end up with a `?` at the end of the url

Reply:
well, that's a matter of you covering for that case

Reply:
Agree, but I'd rather not couple the transformation to query string with adding the prefix that's technically not part of the query string